qubesmanager/appmenu_select.py

```diff
@@ -85,6 +84,8 @@ def save_appmenu_select_changes(self):
             stdin=subprocess.PIPE)
         p.communicate('\n'.join(new_whitelisted).encode())
         if p.returncode != 0:
-            raise RuntimeError('qvm-appmenus --set-whitelist failed')
+            exception_text = QtCore.QCoreApplication.translate(
+                "exception", 'qvm-appmenus --set-whitelist failed')
```

Comment:
Better use format like `Command '{}' failed` and fill the command outside of translated string. Will be easier to translate, and also reusable in other messages.